model/limit.go

```diff
@@ -22,8 +22,10 @@ var (
 )
 
 type Limit struct {
-	Name  string `bson:"_id"`
-	Value uint64 `bson:"value" json:"value"`
+	Id       string `bson:"_id,omitempty" json:"-"`
+	Name     string `bson:"name"`
+	Value    uint64 `bson:"value" json:"value"`
+	TenantID string `json:"tenant_id" bson:"tenant_id"`
```

Comment:
for now I left i, I feel much more comfy with omit empty, as it ensure the mongo will generate an id, otherwise you insert with empty ids and the acceptance tests fail in a way, that took me all Sunday to debug (I actually connected debuggers to both the test and the service and progressed step by step, and yes, inside our testing framework, with containers)